### ISSUE TITLE
Object type looks wrong

### ISSUE THREAD

**Issue by iosware** [basdvries/BDVCountryNameAndCode#4]
You moved dictionary from code into plist file. 
But key "code" in plist is now of type Number which you try to assign to NSString. It definitely is wrong.

**Comment by basdvries** [comment]
You are right. This is bad naming convention. However, memory is only allocated during runtime so it doesn't really matter what the object type says in the plist. I will try to see if I can fix it. I'm also open to pull requests fixing this.
